baker/GrapherBakingUtils.ts

```diff
@@ -58,6 +63,52 @@ export async function getTagToSlugMap(
     return tagsByIdAndName
 }
 
+/**
+ * Returns a map that can resolve Tag names and Tag IDs to whether the tag has any published data insights.
+ * e.g.
+ *   "Women's Rights" -> true
+ *   123 -> true
+ */
+export async function getTagToHasDataInsightsMap(
```

Comment:
Not a big deal, but we could use the native `Set` here, which would contain only tags that have data insights.